bindata/v3.11.0/kube-apiserver/defaultconfig.yaml

```diff
@@ -97,9 +97,9 @@ servicesSubnet: 10.3.0.0/16 # ServiceCIDR
 servingInfo:
   bindAddress: 0.0.0.0:6443
   bindNetwork: tcp4
-  certFile: /etc/kubernetes/static-pod-resources/secrets/serving-cert/tls.crt
+  certFile: /etc/kubernetes/static-pod-resources/secrets/service-network-serving-certkey/tls.crt
```

Comment:
Chose a more-or-less random cert. We would recognize this one as being *for* the KAS, which should help if we get it unexpectedly.  The user-provided value can override this.